tests/test_astronomer_registry_configmap.py

```diff
@@ -0,0 +1,25 @@
+from tests.helm_template_generator import render_chart
+from . import supported_k8s_versions
+import yaml
+import re
```

Comment:
@syamasakigoodrx Great work refactoring the tests! I think we just need one more thing:
```suggestion
import re
import pytest
```